src/PayloadParser.h

```diff
@@ -35,7 +35,7 @@ namespace snowcrash {
     };
 
     /** Request matching regex */
-    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest" SYMBOL_IDENTIFIER "?" MEDIA_TYPE "?[[:blank:]]*";
+    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest([[:blank:]]" SYMBOL_IDENTIFIER ")?" MEDIA_TYPE "?[[:blank:]]*";
```

Comment:
Ah. That's weird. I remember generally allowing `s` at the end, but looks like we have standardised this.